docs/source/markdown/podman-create.1.md

```diff
@@ -22,6 +22,28 @@ Default settings for flags are defined in `containers.conf`. Most settings for
 remote connections use the server's containers.conf, except when documented in
 man pages.
 
+## IMAGE
+
+ The `image` uses a "transport":"details" format.
+
+ Multiple transports are supported when run locally.  Only the `docker` (container registry)
+ transport is allowed for remote access.
+
+  **dir:**_path_
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
+
+  **docker://**_docker-reference_
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$XDG_RUNTIME_DIR/containers/auth.json`, which is set using `(podman login)`. If the authorization state is not found there, `$HOME/.docker/config.json` is checked, which is set using `(docker login)`.
+
+  **docker-archive:**_path_[**:**_docker-reference_]
+  An image is stored in the `docker save` formatted file.  _docker-reference_ is only used when creating such a file, and it must not contain a digest.
+
+  **docker-daemon:**_docker-reference_
+  An image _docker-reference_ stored in the docker daemon internal storage.  _docker-reference_ must contain either a tag or a digest.  Alternatively, when reading images, the format can also be docker-daemon:algo:digest (an image ID).
```

Comment:
The last two sentences seem to be redundant - the format can be a tag, or a digest, or a digest?